.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -12,22 +12,18 @@ labels:
   - error logs
   - stack trace -->
 
-
-
-
-## How to reproduce
+# How to reproduce
```

Comment:
Despite what the linter says, these have to be second-level since the rendered template appears in a page that already has an H1.
```suggestion
## How to reproduce
```